tests/foreman/cli/test_oscap.py

```diff
@@ -57,16 +57,26 @@ def test_positive_list_default_content_with_admin(self, module_target_sat):
             1. Login to shell from admin account.
             2. Execute the scap-content command with list as sub-command.
 
-        :expectedresults: Default scap-content are listed.
+        :expectedresults: Default scap-content are listed. They should already be populated on a freshly installed Satellite.
```

Comment:
with my pedantic hat on, the files on the filesystem should be there, they shouldn't really be imported into satellite on a _completely fresh_ (as in `dnf install -y satellite && satellite-installer`) satellite. Iirc the contents get imported during templatization

Reply:
> I wonder if broker gives me the machine where that has already happened.

I think so, yes. From what I've heard, the machines broker spins up are made from images with installed satellites and scap contents are imported before the image is made.

Reply:
Great, I will rephrase the expected results.